tests/test_registration_embargoes.py

```diff
@@ -8,7 +8,7 @@
 from tests.base import fake, OsfTestCase
 from tests.factories import (
     AuthUserFactory, EmbargoFactory, NodeFactory, ProjectFactory,
-    RegistrationFactory, UserFactory, UnconfirmedUserFactory
+    RegistrationFactory, UserFactory, UnconfirmedUserFactory, DraftRegistrationFactory
```

Comment:
Please try to keep imports ordered where possible (eg alphabetical)- it makes things easier to find later.